packages/connectors/connector-kook/README.md

```diff
@@ -0,0 +1,107 @@
+# KOOK connector
+
+The official Logto connector for KOOK social sign-in web apps.
+
+KOOK 应用社交登录官方 Logto 连接器 [中文文档](#kook-连接器)
+
+**Table of contents**
+- [KOOK connector](#kook-connector)
+  - [Get started](#get-started)
+  - [Create an Application in the KOOK Developer Center](#create-an-application-in-the-kook-developer-center)
+  - [Configure your KOOK connector](#configure-your-kook-connector)
+    - [Config types](#config-types)
+  - [Test KOOK connector](#test-kook-connector)
+  - [Reference](#reference)
+- [KOOK 连接器](#kook-连接器)
+  - [配置你的 KOOK 连接器](#配置你的-kook-连接器)
+    - [配置类型](#配置类型)
+  - [测试 KOOK 连接器](#测试-kook-连接器)
+  - [参考](#参考)
+
+
+## Get started
+
+If you don't know the concept of the connector or don't know how to add this connector to your Sign-in experience, please see [Logto tutorial](https://docs.logto.io/docs/tutorials/get-started/enable-social-sign-in).
```

Comment:
To align with other connectors, we can remove this line.

Reply:
removed

Reply:
removed